router.go

```diff
@@ -119,15 +119,19 @@ func NewRouter(server *Server) *Router {
 		Meta:       make(map[string]any),
 	}
 	router.StatusHandler(&PanicStatusHandler{}, http.StatusInternalServerError)
-	for i := 400; i <= 418; i++ {
+	for i := http.StatusBadRequest; i <= http.StatusTeapot; i++ {
 		router.StatusHandler(&ErrorStatusHandler{}, i)
 	}
+	router.StatusHandler(&RequestErrorStatusHandler{}, http.StatusBadRequest)
 	router.StatusHandler(&ValidationStatusHandler{}, http.StatusUnprocessableEntity)
-	for i := 423; i <= 426; i++ {
+	for i := http.StatusLocked; i <= http.StatusUpgradeRequired; i++ {
 		router.StatusHandler(&ErrorStatusHandler{}, i)
 	}
-	router.StatusHandler(&ErrorStatusHandler{}, 421, 428, 429, 431, 444, 451)
-	router.StatusHandler(&ErrorStatusHandler{}, 501, 502, 503, 504, 505, 506, 507, 508, 510, 511)
+	router.StatusHandler(&ErrorStatusHandler{}, http.StatusMisdirectedRequest, http.StatusPreconditionRequired, http.StatusTooManyRequests, http.StatusRequestHeaderFieldsTooLarge, 444, http.StatusUnavailableForLegalReasons)
```

Comment:
The 444 seems odd.

Is it something special the lib support?

You should add a constant, with a comment, to avoid this magical number here

Reply:
`444 No Response` is an unofficial status code used by nginx. It's unlikely that we would return such errors but removing it would be a breaking change.

I appreciate your attention to details and code smells but I don't want to be excessive. I don't mind a few magical numbers here, especially since it's pretty easy to understand that it's a status code.

Reply:
You put more effort in replying me here, than it would be needed to comment the value in the code 😬

I think a constant with a comment would be better. But a comment without constant is at least expected

Reply:
I agree with both sides. I've added a constant in response.go (as PHP Symfony also does this).

Reply:
Thanks for doing the changes, the code is clearer now